ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2913,18 +2832,6 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-ipi-proxy
-- as: vsphere-ipi-proxy-tp-f14
```

Comment:
Pls also take openshift-openshift-tests-private-release-4.14__amd64-devpreview.yaml into consideration.

Reply:
I wasn't sure if devpreview is ok for us for sure .

Reply:
devpreview is the same as nightly, just use different build from stable channel. It is coped from nightly yaml file and switch over stable build.